tests/registry/test_predicate_mapping.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from eth_abi.registry import (
+    PredicateMapping,
+)
+
+
+@pytest.fixture
+def mapping():
+    return PredicateMapping()
+
+
+def gt(x, y):
```

Comment:
Are you familiar with the [`operator`](https://docs.python.org/3/library/operator.html#operator.gt) module?

Reply:
Yes I am.  I can use it here if you think that makes things more readable.

Reply:
Actually that stuff is going to get taken out since I'm assuming only one argument for predicates.  I had a discussion with @carver and he'd like to use type strings as the canonical form for types.  So that means predicates would only ever receive one string as an argument.  They can parse that string as much as they like in order to figure out if they're interested in it.  We're planning on memoizing the type lookup so the added performance cost of the repeated parsing shouldn't be a problem.  I'm fine with going in that direction.  Thoughts?